docs/usage/9.md

```diff
@@ -0,0 +1,20 @@
+Manual Rollback Procedure
+=========================
+If a migration operation was performed without quiescing the application these steps should not be necessary since the application will be running in place on the source cluster. In a case where a quiesced migration operation was performed and rollback is desired for to failure or some other reason follow the steps below.
```

Comment:
Possible typo?
"rollback is desired for to failure or some other reason follow the steps below"

To:
"..  rollback is desired from a failure or some other reason follow the steps below"

Reply:
Yep, sorry, clearly still having some vision issues. I tried to apply the first correction Derek suggested after restoring the first paragraph and deleted the wrong word.